frontend/src/lib/components/widgets/Selectbox/Selectbox.tsx

```diff
@@ -103,14 +114,22 @@ class Selectbox extends React.PureComponent<Props, State> {
     )
   }
 
-  private onChange = (value: number): void => {
-    this.setState({ value }, () => this.commitWidgetValue({ fromUi: true }))
+  private onChange = (value?: number | undefined): void => {
+    if (value === undefined) {
```

Comment:
Is this `if` actually needed, since `commitWidgetValue` seems also to handle this case of undefined value as well?